### DISCUSSION TITLE
Listbox item double click command

### DISCUSSION THREAD

**Discussion by Giorgi** [AvaloniaUI/Avalonia#6959] [Q&A]
I have a ListBox component and I want to detect when the user double clicks on an item. I saw that there is `DoubleTapped` event but is there a way to trigger a ViewModel command instead?

**Comment by timunie** [comment]
Hi,

Maybe you can add a custom behavior for this, see this link https://wieslawsoltes.github.io/AvaloniaBehaviors/

Happy coding
Tim

**Reply by maxkatz6** [reply]
Or just code behind, which is also a legit option imo. 
But yes, behavior is a nice way to go.

**Reply by timunie** [reply]
Hi @maxkatz6 ,

I think it depends on the needs. If you want to reuse the action in a style I would go the Behavior route. (Can one even add code in a Style?). If it is just for one place code behind may be easier.

BR
Tim
